config/helm/chart/default/templates/Common/csi/daemonset.yaml

```diff
@@ -131,6 +142,19 @@ spec:
       - name: provisioner
         image: {{ include "dynatrace-operator.image" . }}
         imagePullPolicy: Always
+        {{- if eq .Values.debug true }}
```

Comment:
If we adress the debug flag from the `values` file, shouldn't we add a default debug flag value to `values.yaml`? Or am I missing something that I do not get?

Reply:
If it is not set in the values file, helm will use the default value which is `false`. Didn't want to specify it to not confuse the user, but maybe it's a better to add it to the `values.yaml` to be more explicit. - Will add it 👍

Reply:
you don't have to put it into the `values.yaml`, that is just the "default" values for the chart, and we put user-facing flags there

the `debug` flag is not user-facing, so it shouldn't be there
similar to `.Values.manifests`

Reply:
got it, I am also fine with not putting it to the `values` file 👍